scripts/generate_release_post.sh

```diff
@@ -4,7 +4,7 @@ set -eu
 VERSION="$1" # such as 5.3.0-rc.1, 5.0.0
 
 TEMP_VERSION_FILE="$(mktemp -t temp_version_file.XXXX)"
-git for-each-ref --sort=creatordate --format '%(refname)' refs/tags | grep "v[0-9]\.[0-9]\.[0-9]" | grep -v mas | grep -v "v$VERSION" | sed "s/refs\/tags\/v//" > $TEMP_VERSION_FILE
+git for-each-ref --sort=creatordate --format '%(refname)' refs/tags | grep "v[0-9]\+\.[0-9]\+\.[0-9]\+" | grep -v mas | grep -v "v$VERSION" | sed "s/refs\/tags\/v//" > $TEMP_VERSION_FILE
```

Comment:
Unless there's something weird with shell scripts that I'm unaware of, this shouldn't be escaping the plus signs
```suggestion
git for-each-ref --sort=creatordate --format '%(refname)' refs/tags | grep "v[0-9]+\.[0-9]+\.[0-9]+" | grep -v mas | grep -v "v$VERSION" | sed "s/refs\/tags\/v//" > $TEMP_VERSION_FILE
```